docs/sql-data-sources-parquet.md

```diff
@@ -260,6 +260,12 @@ Data source options of Parquet can be set via:
 
 <table class="table">
   <tr><th><b>Property Name</b></th><th><b>Default</b></th><th><b>Meaning</b></th><th><b>Scope</b></th></tr>
+  <tr>
+    <td><code>maxFilesPerTrigger</code></td>
```

Comment:
this is not a Parquet specific option. it's for structured streming only option. we should probably take out